docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,115 +1,135 @@
 ----
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-----
+$ scarb --version
 
-TIP: If you have any trouble installing gmp on your Apple M1 computer, https://github.com/OpenZeppelin/nile/issues/22[here's a list of potential solutions].
+scarb 0.7.0 (58cc88efb 2023-08-23)
+cairo: 2.2.0 (https://crates.io/crates/cairo-lang-compiler/2.2.0)
+sierra: 1.3.0
+----
 
-=== Set up your project
+==== Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create an empty directory, then `cd` into it, and initialize a new scarb project by running:
 
 [,bash]
 ----
-mkdir my-project
-cd my-project
-python3 -m venv env
-source env/bin/activate
+mkdir my_project && cd my_project
+scarb init
 ----
 
-Install the https://github.com/OpenZeppelin/nile[Nile] development environment and then run `init` to kickstart a new project.
-Nile will create the project directory structure and install https://www.cairo-lang.org/docs/quickstart.html[the Cairo language], a https://github.com/Shard-Labs/starknet-devnet/[local network], and a https://docs.pytest.org/en/6.2.x/[testing framework].
+The contents of `my_project` should now look like this:
 
 [,bash]
 ----
-pip install cairo-nile
-nile init
-----
-
-=== Install the library
+$ ls
 
-[,bash]
+Scarb.toml src
 ----
-pip install openzeppelin-cairo-contracts
-----
-
-WARNING: Installing directly through GitHub may contain incomplete or breaking implementations.
-While we aim not to introduce such changes, we still strongly recommend installing through https://github.com/OpenZeppelin/cairo-contracts/releases/[official releases].
 
-=== Use a basic preset
+==== Install the library
 
-Presets are ready-to-use contracts that you can deploy right away.
-They also serve as examples of how to use library modules.
-xref:extensibility.adoc#presets[Read more about presets].
+Add the library to your project be adding the openzeppelin dependency to the `Scarb.toml` file:
 
-[,cairo]
+[,text]
 ----
-// contracts/MyToken.cairo
-
-%lang starknet
-
-from openzeppelin.token.erc20.presets.ERC20 import (
-    constructor,
-    name,
-    symbol,
-    totalSupply,
-    decimals,
-    balanceOf,
-    allowance,
-    transfer,
-    transferFrom,
-    approve,
-    increaseAllowance,
-    decreaseAllowance
-)
+[dependencies]
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0" }
 ----
 
-Compile and deploy it right away:
+WARNING: Make sure the correct tag matching the release is selected.
 
-[,bash]
-----
-nile compile
-
-nile deploy MyToken <name> <symbol> <decimals> <initial_supply> <recipient> --alias my_token
-----
-
-NOTE: `<initial_supply>` is expected to be two integers i.e.
-`1` `0`.
-See xref:utilities.adoc#uint256[uint256] for more information.
+=== Basic usage
 
-=== Write a custom contract using library modules
+Copy the following account contract code into `src/lib.cairo`, and run `scarb build` to compile it:
```

Comment:
It feels like the tutorial ends up somewhat abruptly. If we want to keep it short I'd at least present the following piece of code and then conclude with the commands. Like

```suggestion
This is how it looks to build an account contract using the xref:account.adoc[account module]. Copy the code into `src/lib.cairo`.

(... code ...)

You can now compile it:

---
scarb build
---

```